graph/db/kv_store.go

```diff
@@ -1104,10 +1094,6 @@ func (c *KVStore) addChannelEdge(tx kvdb.RwTx,
 		return ErrEdgeAlreadyExist
 	}
 
-	if c.graphCache != nil {
```

Comment:
We could remove `KVStore.graphCache` and `setGraphCache` too now.

Reply:
it's still used for quite a few PRs. This is just doing part of the work

Reply:
Indeed! Yeah maybe better to separate the move into multiple PRs for quicker iteration.